ips/services/forms.py

```diff
@@ -116,15 +117,40 @@ class ImportPVForm(FlaskForm):
 
 
 class LoadDataForm(FlaskForm):
-    survey_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    shift_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    non_response_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    unsampled_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    tunnel_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    sea_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-    air_file = FileField(validators=[FileRequired(), FileAllowed(['csv'], 'File must be a .csv file.')])
-
+    survey_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    shift_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    non_response_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    unsampled_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    tunnel_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    sea_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+    air_file = FileField(validators=[FileAllowed(['csv'], 'File must be a .csv file.')])
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        fieldnames = [
+            'survey_file', 'shift_file', 'non_response_file', 'unsampled_file',
+            'tunnel_file', 'sea_file', 'air_file' 
+        ]
+    
+        validators = {}    
```

Comment:
Would be useful to leave a comment here explaining what is happening i.e that you're looping through the fields adding a FileRequired validator to the _extra_validator attribute for fields that haven't had a filename set in the meta object.